plugins/Goals/lang/en.json

```
@@ -68,30 +68,30 @@
         "Filename": "filename",
         "GoalConversion": "Goal conversion",
         "GoalConversions": "Goal conversions",
-        "GoalConversionsBy": "Goal %s conversions by type of visit",
-        "GoalIsTriggered": "Goal is triggered",
-        "GoalIsTriggeredWhen": "Goal is triggered when",
+        "GoalConversionsBy": "%s goal conversions by type of visit",
```

The placeholder is replaced with the goals name, so the change makes no sense.
```suggestion
        "GoalConversionsBy": "Goal %s conversions by type of visit",
```